nautobot/dcim/templates/dcim/controller/base.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'generic/object_retrieve.html' %}
+{% load helpers %}
+
+{% block extra_nav_tabs %}
+    {% if object.access_point_groups %}
+        <li role="presentation" {% if active_tab == 'wireless-networks' %} class="active"{% endif %}>
+            <a href="{% url 'dcim:controller_wirelessnetworks' pk=object.pk %}">Wireless-Networks</a>
+        </li>
+        <li role="presentation" {% if active_tab == 'wireless-devices' %} class="active"{% endif %}>
+            <a href="{% url 'dcim:controller_wirelessdevices' pk=object.pk %}">Wireless-Devices</a>
```

Comment:
Should these tabs be

```suggestion
            <a href="{% url 'dcim:controller_wirelessnetworks' pk=object.pk %}">Wireless Networks</a>
        </li>
        <li role="presentation" {% if active_tab == 'wireless-devices' %} class="active"{% endif %}>
            <a href="{% url 'dcim:controller_wirelessdevices' pk=object.pk %}">Wireless Devices</a>
```

Reply:
@cmsirbu recommended the dash, but I can see it done without.